src/databases/MFEM/MFEM.xml

```diff
@@ -1,13 +1,16 @@
 <?xml version="1.0"?>
   <Plugin name="MFEM" type="database" label="MFEM" version="1.0" enabled="true" mdspecificcode="false" engspecificcode="false" onlyengine="false" noengine="false" dbtype="STMD" haswriter="false" hasoptions="false" filePatternsStrict="false" opensWholeDirectory="false">
     <CXXFLAGS>
+      ${CONDUIT_INCLUDE_DIR}
       ${MFEM_INCLUDE_DIR}
       ${VISIT_INCLUDE_DIR}/third_party_builtin
     </CXXFLAGS>
     <LDFLAGS>
+      ${CONDUIT_LIBRARY_DIR}
```

Comment:
If MFEM is built as a static lib (as it is on Windows), then adding CONDUIT_LIBRARY_DIR and CONDUIT_LIB here shouldn't be necessary.  If in fact it is needed, shouldn't it be handled via VISIT_MFEM_LIBDEP in the config-site file?  I found on Windows I still needed the conduit include, but I think that can be handled via a change I propose to make to SetUpThirdParty.cmake to handle dependent includes, and the introduction of VISIT_MFEM_INCDEP to config-site file.